microbench/src/main/java/io/netty/microbench/http2/internal/hpack/DecoderBenchmark.java

```diff
@@ -81,7 +82,7 @@ public void teardown() throws IOException {
 
     @Benchmark
     @BenchmarkMode(Mode.Throughput)
-    public void decode(final Blackhole bh) throws IOException {
+    public void decode(final Blackhole bh) throws IOException, Http2Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: java.io.IOException, io.netty.handler.codec.http2.Http2Exception [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1160)